chains/chain-bridge-hub-kusama/Cargo.toml

```diff
@@ -0,0 +1,40 @@
+[package]
+name = "bp-bridge-hub-kusama"
+description = "Primitives of BridgeHubKusama parachain runtime."
+version = "0.6.0"
+authors.workspace = true
+edition.workspace = true
+license = "GPL-3.0-or-later WITH Classpath-exception-2.0"
+repository.workspace = true
+
+[package.metadata.polkadot-sdk]
+exclude-from-umbrella = true
```

Comment:
This is not needed. It was something specific to polkadot-sdk. The same for the other moved crates.